src/adapter/kuksa.val/kuksa.val.adapter.js

```diff
@@ -149,13 +149,13 @@ class VssAdapter extends Talent {
                 this.subscriptions[absKuksaVssPath].updateAtMs = Math.max(ev.timestamp, this.subscriptions[absKuksaVssPath].updateAtMs);
             }
 
-            this.logger.debug(`Publishing value ${rawValue} to Kuksa.Val at path ${absKuksaVssPath}...`, evtctx);
+            this.logger.debug(`Publishing value ${rawValue} to Kuksa.val at path ${absKuksaVssPath}...`, evtctx);
 
             // Publish raw value to VSS
-            await this.vssSocket.publish(absKuksaVssPath, rawValue);
+            await this.kuksaValSocket.publish(absKuksaVssPath, rawValue);
```

Comment:
maybe we could avoid name changes internal members since there are already in the related name space (e.g. KuksaValAdapter.socket instead of KuksaValAdapter.kuksaValSocket... this will avoid breaking changes for external stuff if they use and reference this member).

Reply:
I agree on the naming. I disagree on the fact, that internal APIs or members should be accessed directly although it works and therefore there should not be any renaming problem.